utils/results/output/resultwriter.go

```diff
@@ -264,6 +264,9 @@ func (rw *ResultsWriter) printTables() (err error) {
 	if err = rw.printJasTablesIfNeeded(tableContent, utils.SastScan, jasutils.Sast); err != nil {
 		return
 	}
+	if err = rw.printJasTablesIfNeeded(tableContent, utils.MaliciousCodeScan, jasutils.MaliciousCode); err != nil {
```

Comment:
I this this will also output empty table for `audit` and the other commands....
please update `IsScanRequested` func so it will reflect that only in the related new command it is requested
(similar to how IaC and Sast only defined for source code and not binary